tests/antithesis/Makefile

```diff
@@ -31,7 +32,7 @@ antithesis-build-etcd-remove-worktree:
 	git worktree remove -f $(BUILDDIR)
```

Comment:
`antithesis-build-etcd-remove-worktree` can be removed

Reply:
Removed.